skatepark/theme.json

```diff
@@ -232,6 +231,11 @@
 					"size": "16px",
 					"slug": "small"
 				},
+				{
+					"name": "Normal",
```

Comment:
We moved Normal out of the theme.json since this isn't a size that is supported by Gutenberg anymore. It should be in custom instead.

Reply:
ugh I read that backwards on the original PR...